helm/release-exporter/templates/deployment.yaml

```diff
@@ -18,20 +18,19 @@ spec:
       containers:
       - name: release-exporter
         image: quay.io/giantswarm/release-exporter:{{ .Chart.Version }}
+        args:
+        - --run-every=30
         resources:
           requests:
             cpu: 100m
             memory: 100Mi
           limits:
             cpu: 200m
             memory: 200Mi
-        env:
-        - name: GITHUB_KEY
-          valueFrom:
-            secretKeyRef:
-              name: release-exporter
-              key: GITHUB_KEY
-              optional: false
+        livenessProbe:
+          httpGet:
+            path: /metrics
+            port: 8000
       - name: grafana-agent
         image: grafana/agent:v0.40.2
```

Comment:
I guess this is for metrics exporting. Have you considered using a servicemonitor instead of a sidecar?

Reply:
Oh so you use it to send data to grafana cloud directly, sure but I would be careful about this use case

Reply:
yeah the main goal here is to have release info from Github in our Grafana cloud. I follow the same approach as [operational-load-exporter](https://github.com/giantswarm/operational-load-exporter/) but I am happy to learn for a better way of achieving this

Reply:
I would think it's fine to do this now :)